packages/ra-core/src/auth/useAuthState.spec.tsx

```diff
@@ -9,23 +9,13 @@ const UseAuth = (authParams: any) => {
     const state = useAuthState(authParams);
     return (
         <div>
-            <span>{state.isLoading && 'LOADING'}</span>
+            <span>{state.isPending && 'LOADING'}</span>
             <span>{state.authenticated && 'AUTHENTICATED'}</span>
         </div>
     );
 };
 
 describe('useAuthState', () => {
-    it('should return a loading state on mount', () => {
```

Comment:
Because by using `placeholderData` the query is not in pending state anymore so this test isn't relevant. We could test the `isLoading` though but I fail to see the point

Reply:
useAuthState is designed to render anonymous content if not authenticated (cf https://marmelab.com/react-admin/useAuthState.html), so I think we still need this test, using `isLoading` and not `isPending`.

Reply:
We probably need a test for the `false` return value